cypress/support/login.js

```diff
@@ -11,13 +12,18 @@ Cypress.Commands.add('login', (username, password) => {
 
   cy.visit('/');
 
-  cy.do([
-    TextField('Username').fillIn(username),
-    TextField('Password').fillIn(password),
-    Button('Log in').click(),
-  ]);
+  // Todo: find the way to wrap interactor to cy chainable object
+  cy.get('input[name=username]', getLongDelay()).should('be.visible');
+  cy.do(TextField('Username').fillIn(username));
+  cy.do(TextField('Password').fillIn(password));
+  cy.do(Button('Log in').click());
```

Comment:
Why it was replaced? Previous verison had chainable cy object and it worked

Reply:
reverted